labels/selector.go

```diff
@@ -36,6 +37,8 @@ type Matcher interface {
 	Name() string
 	// Matches checks whether a value fulfills the constraints.
 	Matches(v string) bool
+	// String returns a human readable matcher.
+	String() string
```

Comment:
I don't see why do we need this method?

Reply:
It makes error output for the unittests readable.

Reply:
yep i get it now, Thanks for clarifying.